mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -7236,6 +7237,34 @@ fn test_gui_storage_coins_functionality() {
     assert_eq!(actual.result, expected);
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_btc_block_header_sync_with_optimzation() {
```

Comment:
Doesn't this test actually check if the coin is activated successfully if SPV optimization config is set?
The test name doesn't match this